src/__tests__/components/fields/image-upload/image-upload.spec.tsx

```diff
@@ -52,7 +53,22 @@ const FrontendEngineWithEventListener = (props: ICustomFrontendEngineProps) => {
 		}
 	}, [eventListener, eventType]);
 
-	return <FrontendEngine {...otherProps} ref={formRef} />;
+	const dispatchDismissReviewModal = () => {
+		formRef.current?.dispatchFieldEvent("dismiss-review-modal", COMPONENT_ID);
+	};
+	const dispatchTriggerSaveReviewImages = () => {
+		formRef.current?.dispatchFieldEvent("trigger-save-review-images", COMPONENT_ID);
+	};
+
+	return (
+		<>
+			<FrontendEngine {...otherProps} ref={formRef} />
+			<Button.Default onClick={dispatchDismissReviewModal}>Dispatch dismiss-review-modal</Button.Default>
+			<Button.Default onClick={dispatchTriggerSaveReviewImages}>
+				Dispatch trigger-save-review-images
+			</Button.Default>
+		</>
+	);
```

Comment:
this will get out of hand as we add more events, can we make it dynamic?

there is helper component call `FrontendEngineWithCustomButton` that gives you access to the ref via `onClick`, you can do a `dispatchFieldEvent` through this